raster/r.series.accumulate/main.c

```diff
@@ -169,8 +169,8 @@ int main(int argc, char *argv[])
     if (G_parser(argc, argv))
         exit(EXIT_FAILURE);
     
-    lo = -1.0 / 0.0;    /* -inf */
-    hi =  1.0 / 0.0;    /* inf */
+	sscanf("-inf", "%lf", &lo);
+	sscanf("inf", "%lf", &hi);
```

Comment:
In #2681, the code was changed to use `-INFINITY` and `INFINITY,` but there's a note that it might not for MSVC (at that time):

```c
    lo = -INFINITY;
    hi = INFINITY;
```